src/pyo3_mock_data/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["maturin>=1.0.0"]
+build-backend = "maturin"
+
+[project]
+name = "prelude-parser"
+requires-python = ">=3.8"
```

Comment:
My toml linter is complaining here, do you know why?

`{"name":"prelude-parser","requires-python":">=3.8"} is not valid under any of the given schemas`

Reply:
This was the even-better-toml extension on VS code, so I'm not sure if this is something invalid.

Reply:
I see why. I copied this from another project of mine and forgot to change the name :man_facepalming:

Reply:
Makes sense to have a toml linter also, I guess! So is prelude-parser a required dependency?